classes/book.rb

```diff
@@ -0,0 +1,10 @@
+# app/classes/book.rb
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] You should implement a method to add rentals in Book class. the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
you can implement it like this:
```ruby
 def add_rental(person, date)
      Rental.new(date, self, person)
  end
```